### PR TITLE
Column meta cache 

### PR DESCRIPTION
**Pull request by ulvii**
Implemented Cache to store Column SQL types and Column Display Sizes, so that PDO SQLSRV does not call ODBC driver for every fetch. This change should improve the driver performance by 15-20% when fetching strings.

### CODE REVIEW

**Comment by v-kaywon** [comment]
Do you want to quickly run Dr. Memory or Valgrind on a simple fetching test to make sure the cache is cleaned up?

**Comment by ulvii** [comment]
@v-kaywon , done.

File: source/shared/core_stmt.cpp [hunk 1]
```diff
@@ -1014,6 +1034,9 @@ void core_sqlsrv_next_result( sqlsrv_stmt* stmt TSRMLS_DC, bool finalize_output_
 
         close_active_stream( stmt TSRMLS_CC );
 
+        //Clear column sql types and sql display sizes.
+        zend_hash_clean( Z_ARRVAL( stmt->col_cache ));	    
+
```
Comment (v-kaywon) [inline source/shared/core_stmt.cpp]: Where does the cache get reinitialized after core_sqlsrv_next_result? The only initialization I see for the col_cache is in the sqlsrv_stmt constructor.

Comment (ulvii) [inline source/shared/core_stmt.cpp]: zend_hash_clean() does not release the buckets, it removes the elements from a hashtable.

**Comment by ulvii** [comment]
#189

**Pull request merged**